examples/tests/sdk_tests/dynamodb-table/global-secondary-index.test.w

```diff
@@ -0,0 +1,100 @@
+bring cloud;
+bring ex;
+bring util;
+
+let api = new cloud.Api();
+
+let table = new ex.DynamodbTable(
+    name: "blog",
+    attributeDefinitions: {
+        type: "S",
+        id: "S",
+        createdAt: "N"
+    },
+    hashKey: "type",
+    rangeKey: "id",
+    globalSecondaryIndex: [{
+        name: "CreatedAtIndex",
+        hashKey: "type",
+        rangeKey: "createdAt",
+        projectionType: "ALL"
+    }],
+);
+
+// Only Hash Key
+let table2 = new ex.DynamodbTable(
+    name: "blog2",
+    attributeDefinitions: {
+        type: "S",
+        id: "S",
+    },
+    hashKey: "type",
+    rangeKey: "id",
+    globalSecondaryIndex: [{
+        name: "TypeIndex",
+        hashKey: "type",
+        projectionType: "ALL"
+    }],
+) as "blog2";
+
+test "Global secondary index" {
+    let idCreated = [
+        {id: "zuegksw", createdAt: 1703412771},
+        {id: "dirnfhw", createdAt: 1703413971},
+        {id: "pdkeruf", createdAt: 1703422171},
+        {id: "azjekfw", createdAt: 1703432071},
```

Comment:
Just a suggestion, if you can make those values easier to estimate by the human eye (shorter for example) it would be great for example {id: "a", createdAt: 10}